app/views/katello/api/v2/content_views/_content_view.json.rabl

```diff
@@ -22,8 +22,14 @@ child :environments => :environments do
   end
 end
 
-child :repositories => :repositories do
-  extends 'katello/api/v2/repositories/show'
+if @object.composite?
+  child :component_repositories => :repositories do
+    extends 'katello/api/v2/repositories/show'
```

Comment:
this will conflict with https://github.com/Katello/katello/pull/5047  that does change it to specific attributes, so be aware of the conflict